src/plugins/saved_objects_management/public/management_section/objects_table/components/table.test.tsx

```diff
@@ -115,6 +116,50 @@ describe('Table', () => {
     expect(component).toMatchSnapshot();
   });
 
+  it('should render gotoApp link correctly for workspace', () => {
+    const item = {
+      id: 'dashboard-1',
+      type: 'dashboard',
+      workspaces: ['ws-1'],
+      attributes: {},
+      references: [],
+      meta: {
+        title: `My-Dashboard-test`,
+        icon: 'indexPatternApp',
+        editUrl: '/management/opensearch-dashboards/objects/savedDashboards/dashboard-1',
+        inAppUrl: {
+          path: '/app/dashboards#/view/dashboard-1',
+          uiCapabilitiesPath: 'dashboard.show',
+        },
+      },
+    };
+    const props = {
+      ...defaultProps,
+      availableWorkspaces: [{ id: 'ws-1', name: 'My workspace' } as WorkspaceAttribute],
+      items: [item],
+    };
+    // not in a workspace
+    let component = shallowWithI18nProvider(<Table {...props} />);
+
+    let table = component.find('EuiBasicTable');
+    let columns = table.prop('columns') as any[];
```

Comment:
`any[]` do we have more specific types here?

Reply:
Yes, updated to specific type `Array<{ render: (id: string, record: unknown) => React.ReactElement }>`